jobbergate-cli/jobbergate_cli/auth.py

```diff
@@ -94,6 +94,29 @@ def validate_token_and_extract_identity(token_set: TokenSet) -> IdentityData:
             log_message=f"Token data could not be extracted to identity: {err}",
             original_error=err,
         )
+
+    if settings.MULTI_TENANCY_ENABLED:
+        try:
+            org_dict = token_data["organization"]
+            JobbergateCliError.require_condition(
+                len(org_dict) == 1,
+                message="Did not find exactly one organization in token payload",
+            )
+            identity.organization_id = list(org_dict.keys())[0]
+        except Exception as err:
+            raise Abort(
+                """
```

Comment:
The error message could be a bit more specific, so we can quickly identify the a possible problem when providing support to end users

Reply:
We don't want to give the user too much information so that it's confusing or troubling. If the command is run in verbose mode, you will see the log_message below displayed. Additionally, Sentry should catch this error and provide more context for us as well.

Reply:
Fair enough